src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator starts working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");

```

Comment:
Similarly, the message 'Excavator stopped working' should also be more informative. Ensure that it clearly indicates that the Excavator has stopped working.